tests/test_evaluable.py

```diff
@@ -209,6 +209,8 @@ def test_diagonalize(self):
           actual=evaluable.diagonalize(self.actual, axis, newaxis))
 
   def test_product(self):
+    if self.desired.dtype == bool:
+      self.skipTest('boolean')
```

Comment:
I overlooked this during first review, but I'm not in favor of using `skipTest` for reasons other than external factors, e.g. having MKL installed or not. I suggest to simply return here.